src/FieldDescription/BaseFieldDescription.php

```diff
@@ -32,33 +32,31 @@
  * Some options are global across the different contexts, other are
  * context specifics.
  *
- * Global options :
- *   - type (m): define the field type (use to tweak the form or the list)
- *   - template (o) : the template used to render the field
- *   - name (o) : the name used (label in the form, title in the list)
- *   - link_parameters (o) : add link parameter to the related Admin class when
- *                           the Admin.generateUrl is called
- *   - accessor : the method or the property path to retrieve the related value
- *   - associated_tostring : (deprecated, use associated_property option)
- *                           the method to retrieve the "string" representation
- *                           of the collection element.
- *   - associated_property : property path to retrieve the "string" representation
```

Comment:
[`associated_property`](https://github.com/sonata-project/SonataAdminBundle/search?q=associated_property) it is used in [`RenderElementExtension`](https://github.com/sonata-project/SonataAdminBundle/blob/2c07756867f2a1a466c5292483cd05e50a1932ff/src/Twig/Extension/RenderElementExtension.php#L263-L269) which means it is used in `render_relation_element` Twig filter that it is used globally: https://github.com/sonata-project/SonataAdminBundle/search?q=render_relation_element